src/code/fv-components/fv-gene-label.js

```diff
@@ -0,0 +1,52 @@
+import React, {PropTypes} from 'react';
+
+const FVGeneLabelView = ({allele, chromosomeDescriptor, chromosomeHeight=122, stripe}) => {
+    var chromStart;
+    Object.keys(BioLogica.Species.Drake.geneList).forEach((geneName) => {
+      const gene = BioLogica.Species.Drake.geneList[geneName];
+      if (gene.alleles.indexOf(allele) > -1) {
+        chromStart = gene.start;
+      }
+    });
+    let bioChromHeight = BioLogica.Species.Drake.chromosomesLength[chromosomeDescriptor.name],
+        stripeHeight = chromosomeHeight * .03,
+        percentHeight = chromStart/bioChromHeight;
```

Comment:
Since this deals so much with BioLogica code pulling it out is a good idea. I ended up keeping the `stripeHeight` calculation in `fv-gene-label`, since it's not actually using the length of the gene from BioLogica (which would be smaller than desired)